src/components/projectDetail/DonateDock.tsx

```
@@ -838,25 +852,30 @@ const DonateDock = () => {
                                   </div>
 
+                                  {/* 모바일: 남은 개수 */}
+                                  <div className="block md:hidden mt-1">
+                                    <span className="text-xs text-sub-gray">{formatRemaining(option.remaining)}</span>
+                                  </div>
+
                                   {isSelected && (
                                     <div className="mt-2 flex items-center">
                                       <button
                                         onClick={(e) => {
                                           e.stopPropagation()
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)